webrender/src/internal_types.rs

```diff
@@ -323,6 +323,8 @@ pub enum ResultMsg<B: hal::Backend> {
         BackendProfileCounters,
     ),
     AppendNotificationRequests(Vec<NotificationRequest>),
+    #[cfg(not(feature = "gleam"))]
+    UpdateWindowSize(DeviceIntSize),
```

Comment:
should this be upstreamed to the current WR?

Reply:
I think no. Because GL handles the change of window dimensions internally. It's only us who need to know if a resize happened.